CMakeLists.txt

```diff
@@ -232,7 +232,7 @@ set(antimicrox_SOURCES
     src/xml/joyaxisxml.cpp
     src/xml/joybuttonslotxml.cpp
     src/xml/joybuttonxml.cpp
-    #src/xml/joydpadxml.cpp
```

Comment:
Interesting, when I was reading CMakeLists.txt I was curious why it was commented, but I didn't wanted to touch it.

Reply:
Yeah, it was because the H file included the CPP file which is AFAIK an antipattern in C-like languages. Here's the official guidelines for template classes: https://isocpp.org/wiki/faq/templates#separate-template-fn-defn-from-decl